sirepo/template/radia.py

```diff
@@ -70,14 +119,24 @@ def background_percent_complete(report, run_dir, is_running):
     )
 
 
+def create_archive(sim):
+    if sim.filename.endswith('dat'):
+        return sirepo.http_reply.gen_file_as_attachment(
+            _dmp_file(sim.id),
+            content_type='application/octet-stream',
+            filename=sim.filename,
+        )
+    return False
+
+
 def extract_report_data(run_dir, sim_in):
     assert sim_in.report in _REPORTS, 'unknown report: {}'.format(sim_in.report)
     if 'reset' in sim_in.report:
         template_common.write_sequential_result({}, run_dir=run_dir)
     if 'geometry' in sim_in.report:
         v_type = sim_in.models.magnetDisplay.viewType
-        f_type = sim_in.models.magnetDisplay.fieldType if v_type == VIEW_TYPE_FIELD\
-            else None
+        f_type = sim_in.models.magnetDisplay.fieldType if v_type ==\
```

Comment:
leave a space before `\` so is more visible.

Reply:
a4e01f3